client/payment-methods-map.js

```diff
@@ -18,6 +18,8 @@ export default {
 		),
 		Icon: CreditCardIcon,
 		currencies: [],
+		capability: 'card_payments',
```

Comment:
This attribute is not used in this PR, but will be used in the overall feature.
Leaving it here, so that I don't have to deal with too many merge conflicts as I split the work in multiple PRs.